onnxruntime/core/framework/tensor_external_data_info.h

```diff
@@ -10,16 +10,23 @@
 namespace onnxruntime {
 class ExternalDataInfo {
  private:
+#ifdef _WIN32
+  using OFFSET_TYPE = int64_t;
```

Comment:
Signed int? Is there any issue if you use uint64_t for all platforms?

Reply:
On Linux, off_t is a signed integer. And, sometimes it's 32 bits, sometimes it's 64 bits. It is very complicated. I'm reading this article: https://android.googlesource.com/platform/bionic/+/master/docs/32-bit-abi.md